test/images/agnhost/pause/pause.go

```diff
@@ -36,21 +36,32 @@ var CmdPause = &cobra.Command{
 
 func pause(cmd *cobra.Command, args []string) {
 	fmt.Println("Paused")
+
 	sigCh := make(chan os.Signal, 1)
-	done := make(chan int, 1)
-	signal.Notify(sigCh, syscall.SIGINT)
-	signal.Notify(sigCh, syscall.SIGTERM)
+	signal.Notify(sigCh, syscall.SIGINT, syscall.SIGTERM)
 	fmt.Println("Signals registered")
-	go func() {
-		sig := <-sigCh
-		switch sig {
-		case syscall.SIGINT:
-			done <- 1
-		case syscall.SIGTERM:
-			done <- 2
-		}
-	}()
-	result := <-done
-	fmt.Printf("exiting %d\n", result)
+	sig := <-sigCh
+
+	// Pods using the agnhost image may choose to override the default
+	// termination message path. They then have to the TERMINATION_MESSAGE_PATH
```

Comment:
Hmm, what purpose does overriding the path serve?

Reply:
The path can be configured in the pod spec. It seemed useful to support changing that default, just in case that someone needs or wants it. That's all, I had no specific use case in mind.

Well, it did allow me do to this as normal user:
```
TERMINATION_MESSAGE_PATH=/tmp/termination.log go run ./test/images/agnhost pause
```